src/main/resources/templates/start.html

```diff
@@ -36,9 +34,7 @@ <h1 class="text-center">Mental Health</h1>
                 </p>
             </main>
         </div>
-
```

Comment:
The empty lines in the code could be removed to have overall cleaner code. This can be applied to multiple files. I feel this would be more in line with Baeldung's source file structure section under clean code standards. https://www.baeldung.com/java-clean-code#3-source-file-structure